src/v2/Apps/Consign/Components/GetPriceEstimate.tsx

```diff
@@ -1,24 +1,114 @@
 import React from "react"
-import { Box, Flex, Text } from "@artsy/palette"
+import {
+  Box,
+  Flex,
+  Image,
+  Input,
+  Join,
+  MagnifyingGlassIcon,
+  Spacer,
+  Text,
+  breakpoints,
+} from "@artsy/palette"
+import { SectionContainer } from "./SectionContainer"
 
 export const GetPriceEstimate: React.FC = () => {
   return (
-    <Flex
-      flexDirection={["column", "row"]}
-      alignItems="center"
-      justifyContent="center"
-    >
-      <Box>
-        <Text variant="title">Get a Price Estimate</Text>
-        <Text variant="text">
-          Discover the potential of reselling an artwork you own.
-        </Text>
-      </Box>
-      <Box>
-        <Box>Artwork</Box>
-        <Box>Artwork</Box>
-        <Box>Artwork</Box>
-      </Box>
-    </Flex>
+    <SectionContainer background="black5">
+      <Flex
+        width="100%"
+        maxWidth={breakpoints.xl}
+        flexDirection={["column", "column", "column", "row"]}
+        justifyContent="center"
+      >
+        <Flex
+          width={["100%", "100%", "100%", "40%"]}
+          flexDirection="column"
+          alignItems={["center", "center", "center", "normal"]}
+          textAlign={["center", "center", "center", "left"]}
+        >
+          <Text variant="largeTitle">Get a Price Estimate</Text>
+          <Spacer my={1} />
+          <Text variant="subtitle" color="black60">
+            Discover the potential of reselling an <br />
+            artwork you own.
+          </Text>
+
+          <Spacer mt={3} />
+
+          {/* TODO: Wire up search bar */}
+          <Flex position="relative" maxWidth={360} width="100%">
+            <Input
+              height={40}
+              placeholder="Tell me the value of my…"
+              style={{ boxShadow: "0px 2px 10px rgba(0,0,0,0.1)" }}
+            />
+            <MagnifyingGlassIcon position="absolute" right="2%" top={1} />
+          </Flex>
+        </Flex>
+
+        <Spacer mb={[4, 4, 4, 0]} />
+
+        <Flex
+          width={["100%", "100%", "100%", "60%"]}
+          alignItems="center"
+          justifyContent="center"
+        >
+          <Join separator={<Spacer mr={5} />}>
+            <ArtworkItem
+              image={
+                <Image
+                  width={165}
+                  height="auto"
+                  src="https://d32dm0rphc51dk.cloudfront.net/3mOXPUFZdK7G6kejFVBSbA/large.jpg"
```

Comment:
Nicole's comps included the artist / work name, so i could search on artsy and copy paste image urls from the artwork pages -- we might want to tune the size a bit at some point tho.

Reply:
Are these links permanent?

Reply:
Not too sure, for now just matching comps, but we can revisit. I'll ping Nicole in slack.